analyzers/src/SonarAnalyzer.Common/Rules/MethodsShouldNotHaveIdenticalImplementationsBase.cs

```diff
@@ -26,20 +26,19 @@
 
 namespace SonarAnalyzer.Rules
 {
-    public abstract class MethodsShouldNotHaveIdenticalImplementationsBase<TMethodDeclarationSyntax, TLanguageKindEnum>
-        : SonarDiagnosticAnalyzer
+    public abstract class MethodsShouldNotHaveIdenticalImplementationsBase<TMethodDeclarationSyntax, TLanguageKindEnum> : SonarDiagnosticAnalyzer
         where TLanguageKindEnum : struct
     {
         protected const string DiagnosticId = "S4144";
         private const string MessageFormat = "Update this method so that its implementation is not identical to '{0}'.";
 
         private readonly DiagnosticDescriptor rule;
-
         protected abstract ILanguageFacade<TLanguageKindEnum> Language { get; }
+        protected abstract TLanguageKindEnum[] RegisteredSyntax { get; }
```

Comment:
I think that we name it SyntaxKinds. Could you please double check?

Reply:
the `RegisteredSyntax` name you mean?

Reply:
Yes.